fastestimator/trace/metric/recall.py

```diff
@@ -28,36 +28,17 @@ class Recall(Trace):
     Args:
         true_key: Name of the keys in the ground truth label in data pipeline.
         pred_key: Name of the keys in predicted label. Defaults to None.
-        labels: The set of labels to include. For more details, please refer to
-            sklearn.metrics.recall_score. Defaults to None.
-        pos_label: The class to report. For more details, please refer to
-            sklearn.metrics.recall_score. Defaults to 1.
-        average: It should be one of {"auto", "binary", "micro", "macro", "weighted", "samples", None}.
-            If "auto", the trace will detect the input data type and choose the right average for you. Otherwise, it
-            will pass its to sklearn.metrics.recall_score. Defaults to "auto".
-        sample_weight: Sample weights, For more details, please refer to
-            sklearn.metrics.recall_score. Defaults to None.
         mode: Restrict the trace to run only on given modes {'train', 'eval', 'test'}. None will always
                     execute. Defaults to 'eval'.
         output_name: Name of the key to store to the state. Defaults to "recall".
     """
     def __init__(self,
                  true_key: str,
-                 pred_key: Optional[str] = None,
-                 labels: Optional[str] = None,
-                 pos_label: Optional[Union[int, str]] = 1,
-                 average: Optional[str] = 'auto',
```

Comment:
why are we dropping all this functionality?

Reply:
same as precision